docs/user-guide/configuration/workflow.md

```diff
@@ -142,5 +144,11 @@ jobs:
             - echo: echo hi
 ```
 
+## Re-running a job's build
```

Comment:
Not sure this belongs in the configuration section of the documentation.
Maybe move it to https://docs.screwdriver.cd/user-guide/FAQ or just have `### Re-running a job's build` as a section under `## Detached Jobs and Pipelines`